File: WorkoutTrackerAPI/Data/Models/UserModels/MuscleSize.cs
namespace WorkoutTrackerAPI.Data.Models.UserModels;

public class MuscleSize : IDbModel
{
    public long Id { get; set; }
    public DateTime Date { get; set; }
    public ModelSize Size { get; set; }

    public long MuscleId { get; set; }
    public Muscle? Muscle { get; set; }

    public string UserId { get; set; } = null!;
    public User? User { get; set; }
}

File: WorkoutTracker/src/app/muscle-sizes/muscle-size.ts
import { DbModel } from "../shared/models/db-model";
import { ModelSize } from "../shared/models/model-size";

export interface MuscleSize extends DbModel {
    date: Date;
    size: ModelSize;
    muscleId: number;
    muscleName: string;
    musclePhoto: string|null;
}

